feature_layers/feature-layer-rendering-mode-map/README.md

```diff
@@ -1,31 +1,34 @@
-# Feature Layer Rendering Mode (Map)
+# Feature layer rendering mode (map)
 
-Render features statically or dynamically.
+Render features statically or dynamically by setting the feature layer rendering mode.
 
-![](FeatureLayerRenderingModeMap.gif)
+![Image of feature layer rendering mode map](FeatureLayerRenderingModeMap.gif)
 
-## How it works
+## Use case
+
+In dynamic rendering mode, features and graphics are stored on the GPU. As a result, dynamic rendering mode is good for moving objects and for maintaining graphical fidelity during extent changes, since individual graphic changes can be efficiently applied directly to the GPU state. This gives the map or scene a seamless look and feel when interacting with it. The number of features and graphics has a direct impact on GPU resources, so large numbers of features or graphics can affect the responsiveness of maps or scenes to user interaction. Ultimately, the number and complexity of features and graphics that can be rendered in dynamic rendering mode is dependent on the power and memory of the device's GPU.
+
+In static rendering mode, features and graphics are rendered only when needed (for example, after an extent change) and offloads a significant portion of the graphical processing onto the CPU. As a result, less work is required by the GPU to draw the graphics, and the GPU can spend its resources on keeping the UI interactive. Use this mode for stationary graphics, complex geometries, and very large numbers of features or graphics. The number of features and graphics has little impact on frame render time, meaning it scales well, and pushes a constant GPU payload. However, rendering updates is CPU and system memory intensive, which can have an impact on device battery life.
+
+## How to use the sample
 
-To change `FeatureLayer.RenderingMode` using `LoadSettings`:
+Use the 'Animated Zoom' button to trigger the same zoom animation on both static and dynamically maps.
 
-1. Create a `ArcGISMap`.
-2. Set preferred rendering mode to map, `mapBottom.getLoadSettings().setPreferredPointFeatureRenderingMode(FeatureLayer.RenderingMode.DYNAMIC)`.
-    * Can set preferred rendering mode for `Points`, `Polylines`, or `Polygons`.
-    * `Multipoint` preferred rendering mode is the same as point.
-3. Set map to `MapView`, `mapViewBottom.setMap(mapBottom)`.
-4. Create a `ServiceFeatureTable` from a point service, `new ServiceFeatureTable("http://sampleserver6.arcgisonline.com/arcgis/rest/services/Energy/Geology/FeatureServer/0");`.
-5. Create `FeatureLayer` from table, `new FeatureLayer(poinServiceFeatureTable)`.
-6. Add layer to map, `mapBottom.getOperationalLayers().add(pointFeatureLayer.copy())`
-    * Now the point layer will be rendered dynamically to map view.
+## How it works
+
+1. Create an `ArcGISMap` and call `getLoadSettings()` and then `setPreferred[Point/Polyline/Polygon]FeatureRenderingMode(...)`.
```

Comment:
I prefer our original instructions to make the API read clearer: 

2. Set preferred rendering mode to map, `mapBottom.getLoadSettings().setPreferredPointFeatureRenderingMode(FeatureLayer.RenderingMode.DYNAMIC)`.	
    * Can set preferred rendering mode for `Points`, `Polylines`, or `Polygons`.

Reply:
I've had a go at restructuring this section a little. Let me know what you think?